README.md

```diff
@@ -150,3 +150,28 @@ When running the app locally, ensure you are using the development `GOOGLE_SHEET
 (See google drive for this)
 
 If updating your environment variable doesn't work, try `rm -rf _build` and then starting your server again
+
+### Image uploads
+
+##### File size
+
+The Wagtail cms allows for a maximum file size of 10MB.
```

Comment:
I would suggest to add a recommendation for the file size, for example: 1mb max if possible